nixos/modules/services/networking/hylafax/options.nix

```diff
@@ -7,6 +7,7 @@
 
 let
 
+  inherit (lib.meta) getExe';
   inherit (lib.options) literalExpression mkEnableOption mkOption;
   inherit (lib.types)
```

Comment:
Sorting this by category is not useful

Reply:
I have now added a commit that merges all those `inherit` statements (up to those from `lib.types` which are not included in the `lib` namespace).